### PR TITLE
refactor(blade): rebrand progressbar

### PR DESCRIPTION
**Pull request by anuraghazra**

### CODE REVIEW

File: packages/blade/src/components/ProgressBar/ProgressBar.tsx [hunk 1]
```diff
@@ -22,11 +22,6 @@ type ProgressBarCommonProps = {
    * Sets aria-label to help users know what the progress bar is for. Default value is the same as the `label` passed.
    */
   accessibilityLabel?: string;
-  /**
-   * Sets the contrast for the progress bar
-   * @default 'low'
-   */
-  contrast?: ColorContrastTypes;
```
Comment (kamleshchandnani) [inline packages/blade/src/components/ProgressBar/ProgressBar.tsx:29]: can you add this to the migration doc?

File: packages/blade/src/components/ProgressBar/ProgressBar.tsx [hunk 2]
```diff
@@ -105,7 +100,6 @@ const progressBarHeight: Record<NonNullable<ProgressBarCommonProps['size']>, 2 |
 
 const ProgressBar = ({
   accessibilityLabel,
-  contrast = 'low',
   intent,
```
Comment (kamleshchandnani) [inline packages/blade/src/components/ProgressBar/ProgressBar.tsx:103]: we wanted to get rid of intent prop right?

File: packages/blade/src/components/ProgressBar/ProgressBar.tsx (hunk 1 above)
Comment (anuraghazra) [reply packages/blade/src/components/ProgressBar/ProgressBar.tsx:29]: done

File: packages/blade/src/components/ProgressBar/ProgressBar.tsx (hunk 2 above)
Comment (anuraghazra) [reply packages/blade/src/components/ProgressBar/ProgressBar.tsx:103]: No 🤔, intent will still be there right? We want to have colored progressbars also

**Review by kamleshchandnani — APPROVED** [review]

**Pull request merged**